mlir/test/Dialect/Rock/integration/reduce/reduce_max/rock-reduce-max-case4.mlir

```diff
@@ -0,0 +1,36 @@
+// This test is checking for larger reductions with larger block and grid sizes
+
+// RUN: sed -e 's/#arch/%arch/g; s/#features/%features/g' %s | rocmlir-gen -ph -print-results -fut test_reduce -verifier clone - | rocmlir-driver -host-pipeline xmodel -kernel-pipeline full | xmir-runner --shared-libs=%linalg_test_lib_dir/libmlir_rocm_runtime%shlibext,%conv_validation_wrapper_library_dir/libconv-validation-wrappers%shlibext,%linalg_test_lib_dir/libmlir_runner_utils%shlibext,%linalg_test_lib_dir/libmlir_c_runner_utils%shlibext --entry-point-result=void | FileCheck %s --check-prefix=CLONE
```

Comment:
I'd go for a rather stronger `sed` token, something like `##ARCH_SUBST##`

Reply:
done!